Modules/Libs/QuestieHash.lua

```diff
@@ -57,7 +64,8 @@ end
 
 function QuestieHash:AddNewQuestHash(questId)
     Questie:Debug(Questie.DEBUG_DEVELOP, "[QuestieHash] AddNewQuestHash:", questId)
-    questLogHashes[questId] = QuestieHash:GetQuestHash(questId)
+    local oldHash = questLogHashes[questId]
+    questLogHashes[questId] = QuestieHash:GetQuestHash(questId) or oldHash
```

Comment:
oldHash is most likely nil --> if there is cachemiss, then the whole quest is missing from questLogHashes again